timeline-bubbles/D3/script.js

```diff
@@ -19,9 +23,10 @@ const margin = { top: 40, right: 100, bottom: 30, left: 20 },
     margin.top -
     margin.bottom;
 
-// Clean up before drawing
-// By brutally emptying all HTML from plot container div
-d3.select('#times-timeline').html('');
+(config.yTranslation = isMobile ? height / 4 : height / 2),
```

Comment:
What is going on here :mystery:

Reply:
Weirdest `prettier` formatting I've seen